SUSE/rmt
# Add missing repos details

## Description
When registering against RMT with suseconnect, missing repos are not mentioned in the error message

**How to test**
Delete any mirrors prior to starting this test.
```bash
> # run rmt server in a terminal
> bin/rails s
> # in another shell, run
> docker run --rm -it --network host registry.suse.com/suse/sle15 bash
> (container) zypper in -y suseconnect-ng
> (container) suseconnect --url http://172.17.0.1:4224
> # This should throw an error mentioning the missing repository ids 
```

Fixes #1039 

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.
